lib/campaign.js

```diff
@@ -36,8 +36,14 @@ class Campaign {
   }
 
   getOne(id, appId, cb) {
-    if (!id || typeof id === 'function') {
-      return cb(new Error('id parameter must be provided.'))
+    if (typeof id === 'function') {
+      return id(
```

Comment:
Same question here about returning a promise? Should we do that and then call the cb?

Reply:
If it were only me, I'd drop support for callback altogether :) Not sure what percentage of the user base uses them.

Reply:
If there's interest in dropping support for the callbacks, you could deprecate them and add a warning, giving the community time to respond to the change and possibly advocate for its continued support. We did something similar in the [ruby project](https://github.com/adimichele/hubspot-ruby/pull/148)

Reply:
Keeping the callbacks for now but I think it's worth discussing removing them wholesale!